packages/frontend/src/providers/Tracking/context.ts

```diff
@@ -0,0 +1,6 @@
+import { createContext } from 'react';
+import { type TrackingContextType } from './types';
+
+const TrackingContext = createContext<TrackingContextType>(undefined as any);
```

Comment:
```suggestion
const TrackingContext = createContext<TrackingContextType | undefined>(undefined);
```

this?

Reply:
I am going to avoid making the change, as it triggers an error in two other files. In this PR, I am trying to avoid any changes that are out of scope.

Reply:
cool!